meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_list.html

```diff
@@ -26,11 +26,11 @@
             <form class="" action="{{ request.path }}" method="post">
                 <div class="form-group">
                     {% csrf_token %}
-                    <div class="widget widget--{{ form.token|widget_type }}">
+                    <div class="widget widget--{{ form.token|widget_type }} widget--vote">
```

Comment:
@sabinammm ok styling was more annoying then complicated but I had to add a class name here as there is nothing rendered from `widget_type` don't know why though?